packages/core/docs/reference/core.md

```diff
@@ -3,6 +3,13 @@
 | Name   | Type | Required | Description |
 |--------|------|----------|-------------|
 | `name` | `string` | `true` | Service name |
+| `maintenanceMode` | `bool` | `false` | Maintenance mode on/off |
+
+## Name
+Service name setting is used as a route key between microservices.
```

Comment:
Are you sure? I thought it was defined in the route config `route.prefix`

Reply:
@BrRenat my bad